packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -554,6 +555,22 @@ class _AppBarState extends State<AppBar> {
         overflow: TextOverflow.ellipsis,
         child: title,
       );
+
+      // Set maximum text scale factor to [_kMaxTitleTextScaleFactor] for the
```

Comment:
needs ///

Reply:
This is a comment though and not documentation. If we want it to be documentation it would have to be moved somewhere else.

Reply:
NVM, forgot about https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use--for-public-quality-private-documentation